website/src/components/Showcase/Showcase.js

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect } from 'react';
+import Card from './Card';
+import styles from '../../pages/showcase.module.css';
+// import cardList from './AppsList';
+import cardList from '../../../showcase.json';
+
+function Showcase() {
+  useEffect(() => {
```

Comment:
Incorrect place to use. You should use it not in component but on your page.
Use `@theme/Head` instead of `useEffect`

Reply:
I read through the docs you shared in your notes to understand the use case for useEffect and also thanks for the link to the docusaurus docs to see the usage of @theme/Head